### PR TITLE
TAC Minutes 2023-09-21

### PR DESCRIPTION
**Pull request by helen-lau**

### CODE REVIEW

File: TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+
+**2023 TAC Goals: Cross project collab survey.**
+
+
+* Eric Voit (EV) gave an update on the cross project collab survey that was distributed to project mentors. 
+* The goal is to drive project mentor interest on shared goals and code dependencies within their projects. There have been minimal responses to the survey from project mentors. 
+
+
+*** Action item to all Project Mentors: Take the cross project collab survey. ***
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md:150]: I thought the action item was for the mentors to get the projects to respond to the survey (as opposed to the mentors responding to the survey)

File: TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+
+
+* DM brought up that this is the time of the year where the board reviews the charter and makes changes. Please review https://github.com/confidential-computing/governance/pull/185.
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md:153]: Add: DM has an action item to report this proposed charter tweak to the board

File: TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+
+
+**CISA RFI: working session**
+
+
+* Working doc: https://docs.google.com/document/d/1hEZZfsmMtBpOYkCpRTMNYFbY9_qEYK_VxMRlXhQxfEU/edit?usp=sharing
+
+
+*** Action item: Alec Fernandez to take a look at Terminology whitepaper (https://confidentialcomputing.io/wp-content/uploads/sites/10/2023/03/Common-Terminology-for-Confidential-Computing.pdf) for repurposable text in two places in the RFI.***
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md:165]: I think both the Terminology and the Technical Analysis whitepaper were mentioned.  And ideally _reference_ not _repurpose_ text.

File: TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md (hunk 1 above)
Comment (helen-lau) [reply TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md:150]: Hi Dave. I followed up with Lily following the TAC last week, and she sent it to the project mentors. Unless Lily advises that she'd like to send it to the projects for a response, I'll leave this action item as is for the minutes.

Comment (dthaler) [reply TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md:150]: That's not my point.

```suggestion
*** Action item to all Project Mentors: Get the projects to take the cross project collab survey. ***

```

File: TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md (hunk 3 above)
Comment (helen-lau) [reply TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md]: I amended the text. Tweak if needed here too

File: TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md (hunk 2 above)
Comment (dcmiddle) [reply TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md:154]: FYI I took care of this after the meeting by sending a note to the board. I'll mention this in action item updates in the next meeting 2023/10/05

File: TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+Proposed: That the minutes of the September 7, 2023 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+
+
+
+** Congratulations to rust-spdm**
+
+
+* TAC Voting members voted to approve rust-spdm as a new project.
```
Comment (dcmiddle) [inline TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md:127]:
```suggestion
* TAC Voting members voted by email to approve rust-spdm as a new project.
  https://lists.confidentialcomputing.io/g/tac/topic/101330316#1110
```

File: TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+* Chris Ramming - VMWare: Common developer API for confidential computing. VMWare will introduce a project to the TAC mailing list in two weeks.
+
+
+
+
+**Meeting Minute Approval**
+
+
+Proposed: That the minutes of the September 7, 2023 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
```
Comment (dcmiddle) [inline TAC/Meetings/2023/2023-09-21/TAC Minutes 2023-09-21.md:120]:
```suggestion
* DM reviewed the prior meeting minutes
* Quorum was reached and the minutes were approved by the voting members of the TAC
```

**Comment by dcmiddle** [comment]
Not a substantive correction, but @helen-lau could you also please remove the extra newlines.

**Comment by helen-lau** [comment]
> Not a substantive correction, but @helen-lau could you also please remove the extra newlines.

I got rid of the extra lines

**Review by dcmiddle — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

**Pull request merged**